build.xml

```diff
@@ -563,7 +563,7 @@
           <dependency groupId="org.hdrhistogram" artifactId="HdrHistogram" version="2.1.9"/>
           <dependency groupId="commons-cli" artifactId="commons-cli" version="1.1"/>
           <dependency groupId="commons-codec" artifactId="commons-codec" version="1.9"/>
-          <dependency groupId="org.apache.commons" artifactId="commons-lang3" version="3.1"/>
+          <dependency groupId="org.apache.commons" artifactId="commons-lang3" version="3.11"/>
```

Comment:
I forgot to ask about this before, why did the library version have to bump?

Reply:
recursive toString... they didn't support in 3.1, they added in 3.2... felt going to latest was best if bumping...